src/main/java/com/airlines/entity/Crew.java

```diff
@@ -0,0 +1,42 @@
+package com.airlines.entity;
+
+public class Crew {
+    private final int id;
+    private final String name;
+
+    private Crew(Builder builder) {
```

Comment:
I guess we can remove builder here and use just plain constructor, because we have there only two fields

Reply:
fixed in 83169970bdc295fe755569bd150d7073424f7b44